components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/cache/CacheEntry.java

```diff
@@ -22,7 +22,7 @@
  * Represents a cache entry. Any value that needs to be included in the
  * cache must be encapsulated with a class deriving from this class.
  */
-public abstract class CacheEntry extends org.wso2.carbon.identity.application.common.cache.CacheEntry {
+public abstract class CacheEntry extends org.wso2.carbon.identity.core.cache.CacheEntry {
```

Comment:
```
public abstract class CacheEntry extends CacheEntry {
```
Can't we write like this by importing `org.wso2.carbon.identity.core.cache.CacheEntry`?

Reply:
We can't import like this way since CacheEntry is already defined in this compilation unit.